extensions/wikia/AdEngine/js/lookup/rubiconFastlane.js

```diff
@@ -0,0 +1,175 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.rubiconFastlane', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.adTracker',
+	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, adTracker, adLogicZoneParams, doc, log, win) {
+	'use strict';
+
+	var called = false,
+		config = {
+			oasis: {
+				TOP_LEADERBOARD: [[728, 90],[970, 250]],
+				TOP_RIGHT_BOXAD: [[300, 250],[300, 600]],
+				LEFT_SKYSCRAPER_2: [[160,600]],
+				LEFT_SKYSCRAPER_3: [[160,600]],
+				INCONTENT_BOXAD_1: [[300, 250]],
+				PREFOOTER_LEFT_BOXAD: [[300, 250]],
+				PREFOOTER_RIGHT_BOXAD: [[300, 250]]
+			},
+			mercury: {
+				MOBILE_IN_CONTENT: [[300, 250]],
+				MOBILE_PREFOOTER: [[300, 250]],
+				MOBILE_TOP_LEADERBOARD: [[320, 50]]
```

Comment:
Me nitpicking: can we decide to keep a whitespace or not after comma? :japanese_ogre: